assets/css/tablesorter.theme.glotpress.css

```diff
@@ -19,6 +19,7 @@ GlotPress Theme
 	cursor: pointer;
 	text-align: center;
 	border: 1px solid #eee;
+	padding-right: 20px !important;
```

Comment:
Yes, but it's not very good CSS.  We first set the padding correctly with the tablesorter css, then override it with the table css and then set it back to what it should have been in the first place with yet another selector.

Using `!important` keeps it logically simpler and all of the css associated with the tablsorter in a single file instead of spreading it out across multiple.